### PR TITLE
Item type terms

### PR DESCRIPTION
**Pull request by denismaier**
## Description

Some style require descriptive terms added for certain item types. This poses challenges for localization. This adds a term for every item type to `csl-terms.rnc`.

Fixes https://github.com/citation-style-language/schema/issues/178

See also https://github.com/citation-style-language/zotero-bits/issues/70

## Type of change

Please delete options that are not relevant.

- [X] Term addition (adds a new terms
- [X] This change requires a documentation update

## Checklist

- [X] I have installed the repo [pre-commit hook](https://github.com/citation-style-language/schema/blob/master/CONTRIBUTING.md#pre-commit-hooks); if not, and I have modified any of the schema files, I have run trang and/or prettier on the files, per [CONTRIBUTING](https://github.com/citation-style-language/schema/blob/master/CONTRIBUTING.md)
- [X] I have performed a self-review of my own code
- [X] I have included suggested corresponding changes to the documentation (if relevant)

### CODE REVIEW

File: schemas/styles/csl-terms.rnc [hunk 1]
```diff
@@ -7,11 +7,14 @@ div {
     terms.gender-assignable
     | terms.gender-variants
     | terms.locator
-    | 
+    |
+      ## Item types
+      item-types
```
Comment (bdarcus) [inline schemas/styles/csl-terms.rnc]: I don't think there's any benefit to adding the doc string there.

```suggestion
    | item-types
```

Comment (denismaier) [reply schemas/styles/csl-terms.rnc]: Done

**Comment by bdarcus** [comment]
This seems a simple and uncontroversial change. Ready to merge.

Any other feedback?

**Comment by bwiernik** [comment]
We had also discussed adding `type` as a special term where `<text term="type"/>` would render the term corresponding to the item's `type` (e.g., Map for a `map` item, Journal Article for an `article-journal`). @cormacrelf suggested that it would be a miniscule change for processors to add such a hash table and that it could go into 1.0.2.

**Comment by denismaier** [comment]
> We had also discussed adding `type` as a special term where `<text term="type"/>` would render the term corresponding to the item's `type` (e.g., Map for a `map` item, Journal Article for an `article-journal`). @cormacrelf suggested that it would be a miniscule change for processors to add such a hash table and that it could go into 1.0.2.

If I remember correctly, we've agreed to do this in two steps. First terms, lookup logic later.
(I'll add another PR later or tomorrow with target 1.1)

**Comment by bwiernik** [comment]
That's fine by me.

**Comment by denismaier** [comment]
Ok. So how do we proceed. I'd prepare the next pr once this one is merged. Or can I already do that now? (Not sure how the branching strategy is supposed to work in this case.)

**Comment by bwiernik** [comment]
You can open a pull request against v1.1

**Comment by denismaier** [comment]
Hmm, maybe the extended logic will be covered by https://github.com/citation-style-language/schema/issues/240
So, actually we won't need a special PR for this. WDYT?

**Comment by bwiernik** [comment]
No, these are unrelated. `<term = "type"/>` is something that would be written in a style.

**Comment by denismaier** [comment]
Ok. Got it.

**Comment by bdarcus** [comment]
so this is thus ready to merge?

**Comment by denismaier** [comment]
Yes. It's ready to be merged. I'll open another issue with target 1.1 later. That will cover the remains bit.

**Pull request merged**